rrfs-test/validated_yamls/templates/obtype_config/gnss_zenithTotalDelay.yaml

```diff
@@ -0,0 +1,94 @@
+     - obs space:
+         name: Ground GNSS ZTD  
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 300e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: "data/obs/ioda_ztd.nc"
+
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: jdiag_out_ztd.nc
+
+         simulated variables: [zenithTotalDelay]
+         observed variables: [zenithTotalDelay]
+
+       obs operator:
+         name: GroundgnssMetOffice
+       linear obs operator:
+         name: GroundgnssMetOffice
+
+         observation alias file: obsop_name_map.yaml
+
+       obs error:
+         covariance model: diagonal
+
+       obs localizations:
+         - localization method: Horizontal Gaspari-Cohn
+           lengthscale: 300e3
+
+       obs filters:
+       # Pre-online regional domain check (latitude) for CONUS
+       - filter: Domain Check
+         apply at iterations: 0,1   #pre-filters(0,1), post(2)
+         where:
+           - variable:
+               name: MetaData/latitude
+             minvalue:  20.0
+             maxvalue:  60.0
+         action: 
+            name: reduce obs space
+
+       # Pre-online regional domain check (longitude) for CONUS
+       - filter: Domain Check
+         apply at iterations: 0,1
+         where:
+           - variable:
+               name: MetaData/longitude
+             minvalue: -135.0
+             maxvalue: -55.0
+         action: 
+            name: reduce obs space
```

Comment:
Lines 34-54 can be combined into one section.
See https://github.com/NOAA-EMC/RDASApp/issues/315
example:
```
filter: Domain Check
where:
- variable:
    name: MetaData/latitude
  minvalue: -9.06518
  maxvalue: 11.4725
- variable:
    name: MetaData/longitude
  minvalue: 91.7506
  maxvalue: 112.268
action:
  name: reduce obs space
```